config/routes.rb

```diff
@@ -4,7 +4,10 @@
   get '/login', to: 'sessions#new'
   post '/login', to: 'sessions#create'
   delete '/logout', to: 'sessions#destroy'
+  get '/cart', to: 'carts#show'
+  post '/cart_items', to: 'cart_items#create'
   resources :users
   resources :home_page
   resources :dishes
+  resources :cart_items
```

Comment:
you could do
```ruby
resources :cart_items, only: :create
```
and this `post '/cart_items', to: 'cart_items#create'` is not needed anymore